app/experimenter/static/rapid/components/experiments/ExperimentDetails.tsx

```diff
@@ -169,6 +170,21 @@ const ExperimentDetails: React.FC = () => {
     );
   }
 
+  let rejectFeedback;
+
+  if (experimentData.reject_feedback) {
+    const messageDate = new Date(experimentData.reject_feedback.changed_on);
+    rejectFeedback = (
+      <>
+        <h3 className="my-4">Review Feedback</h3>
+        <div className="alert alert-secondary" role="alert">
+          <p className="font-weight-bold"> {messageDate.toDateString()}</p>
+          <p>Reject reason: {experimentData.reject_feedback.message}</p>
```

Comment:
Is the expectation that we'd always have a single line of feedback from Remote Settings?
Also, a bit extreme, but do we need to make sure this is sanitized at all since it's coming from Remote Settings and we aren't going to XSS ourselves or something? (Although I guess if somebody already has access to our Remote Settings bucket, having them return a reject reason with potential `<script>` tags is probably the least of our worries)

Reply:
Both fine points but seems to support multiple lines and properly escape characters:

![Screenshot_2020-07-30 Mozilla Experimenter(6)](https://user-images.githubusercontent.com/119884/88976714-83fdf580-d28a-11ea-8cd1-a48023344991.png)